chapters/3-package-information.md

```diff
@@ -946,56 +946,59 @@ Example:
 
 **3.21.3** Cardinality: Optional (one or many)
 
-**3.21.4** Data Format: `<category> <type> <locator>`
+**3.21.4** Data Format: `<category> <URI>` <a name="3.21.4"></a>
 
```

Comment:
Even _if_ we had agreed on removing "type", it should first to remain as a deprecated field for one version of the spec. But I don't believed we had agreed on deprecating it. Have we?

Reply:
> The term URI here can be confusing in an RDF/linked data context, because almost everything there is a URI.

How about `<external-URI>`?  Or `<reference-URI>`?  Or just `<reference>` and leave the URI-ness to the subsequent text?

> Even if we had agreed on removing "type", it should first to remain as a deprecated field for one version of the spec. 

That would be possible.  The 2.1 wording [was][1]:

    <type> is an [idstring] that is defined in Appendix.

The Appendix reference was fixed in 02aaac402b, but that hasn't been released yet.  So I don't think a breaking change with such an underspecified field is a big deal, but if we want to keep support through 3.0 (or even 4.0), I'd be ok with:

```markdown
Data Format: `<category> [<type>] <URI-or-whatever>`
```

> But I don't believed we had agreed on deprecating it. Have we?

This PR is just my recommendations.  Review and approval depends on whether we end up agreeing or not, but agreement doesn't have to be established before the PR is filed ;).

[1]: https://spdx.org/spdx-specification-21-web-version#h.7i829o8ausxy